feat(oiiotool): oiiotool --layersplit, new command to split layers (#4591)
AcademySoftwareFoundation/OpenImageIO: src/doc/oiiotool.rst

    image comprised of the subimages of all original images appended
    together.

.. option:: --layersplit

>     Remove the top image from the stack, split it into its consituent
>     channel-name-based layers, and push them all onto the stack
>     (first to last).

Reviewer: ```suggestion
    Remove the top image from the stack, split it into a separate image for
    each of its constituentchannel-name-based layers, and push them all
    onto the stack (first to last).
```

Reviewer: A spelling fix, and also added a clause to clarify that you are splitting into multiple images, not just subimages or whatever.

Author: Updated, thanks !

    By "layer" we mean a subset of the initial channels which, when named
    using the convention "LAYERNAME.channelname", all share the same layer
    name.

> 

Reviewer: Do you want to clarify what should happen to the channels that don't have any '.' at all? Are they gathered into one image together? Is it guaranteed to be the first image?

Author: Yes and yes ! (I've just checked the exr input code just to make sure)
I've updated the docs :)
